homeassistant/components/qnap/sensor.py

```diff
@@ -42,92 +42,91 @@
 _SYSTEM_MON_COND: tuple[SensorEntityDescription, ...] = (
     SensorEntityDescription(
         key="status",
+        translation_key="status",
         name="Status",
-        icon="mdi:checkbox-marked-circle-outline",
     ),
     SensorEntityDescription(
         key="system_temp",
         name="System Temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
-        icon="mdi:thermometer",
         state_class=SensorStateClass.MEASUREMENT,
     ),
 )
 _CPU_MON_COND: tuple[SensorEntityDescription, ...] = (
     SensorEntityDescription(
         key="cpu_temp",
+        translation_key="cpu_temp",
         name="CPU Temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
-        icon="mdi:checkbox-marked-circle-outline",
```

Comment:
This should be the default temperature icon, will change that in a follow-up PR.